trade-finance-manager-api/src/v1/rest-mappings/mappings/facilities/mapFacilityType.js

```diff
@@ -1,4 +1,4 @@
-const { FACILITY_TYPE } = require('@ukef/dtfs2-common');
+const { FACILITY_TYPE, FACILITY_TYPE_MAPPED } = require('@ukef/dtfs2-common');
 const CONSTANTS = require('../../../../constants');
 
 const mapFacilityType = (facility) => {
```

Comment:
Could add the return type from our lovely commonised bit in some JSDOC but no big deal

Reply:
I've added as much js doc as I dare. I do not know why we map to such odd types (when they aren't consistent with the front end types either...)

Reply:
Bump

Reply:
I dislike this as well. The mapFacility function is incredibly surprising, and adding documentation demonstrates such. I think it might be worth us catching up in a call about this tomorrow -- as this should be simplifying matters.